src/lib/models/change-reason.js

```diff
@@ -35,6 +35,15 @@ class ChangeReason extends Model {
     validators.assertEnum(type, changeReasonTypes);
     this._type = type;
   }
+
+  get isEnabledForNewChargeVersions () {
+    return this._isEnabledForNewChargeVersions;
+  }
+
+  set isEnabledForNewChargeVersions (value) {
+    validators.assertIsBoolean(value);
```

Comment:
Do you think we set a default value of true in the constructor then we don't have to worry about setting this value for new charge versions. I assume it will be very unlikely that we would want to create a new charge version that is disabled.